### PR TITLE
[llava][22/N] Turn on xnnpack shared workspace and add README.md

### PR DESCRIPTION
**Pull request by larryliu0820**
Summary: As titled.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: examples/models/llava/README.md [hunk 1]
```diff
@@ -1,20 +1,88 @@
...
+
+### Build C++ runner
+
+Run the following cmake commands from `executorch/`:
+
+```bash
+# build libraries
+cmake                                               \
+    -DCMAKE_INSTALL_PREFIX=cmake-out                \
+    -DCMAKE_BUILD_TYPE=Debug                        \
```
Comment (iseeyuan) [inline examples/models/llava/README.md:36]: Any specific reason for debug? Would it affect the runtime speed?

File: examples/models/llava/README.md [hunk 2]
```diff
@@ -1,20 +1,88 @@
 ## Summary
-In this example, we initiate the process of running multi modality through ExecuTorch.
-- Demonstrate how to export the image encoder model in the [LLava](https://github.com/haotian-liu/LLaVA) multimodal model.
-- Provide TODO steps on how to use the exported .pte file and the existing [exported Llama2 model](https://github.com/pytorch/executorch/tree/main/examples/models/llama2), to build the multimodal pipeline.
+LLaVA is the first multi-modal LLM ExecuTorch supports. In this directory, we
+- Host a model definition for LLaVA.
+- Demonstrate how to export [LLavA](https://github.com/haotian-liu/LLaVA) multimodal model to a .pte file.
+- Provide a C++ runner that loads the .pte file, the tokenizer and an image, then generate responses based on user prompt.
```
Comment (iseeyuan) [inline examples/models/llava/README.md:5]: Where is the image from? Can user generate image.pt from there own images?

**Review by iseeyuan — APPROVED** [review]

File: examples/models/llava/README.md (hunk 2 above)
Comment (larryliu0820) [reply examples/models/llava/README.md:5]: Will add a util in next PR

File: examples/models/llava/README.md (hunk 1 above)
Comment (larryliu0820) [reply examples/models/llava/README.md:36]: Need some testing on release build.

**Review by digantdesai — APPROVED** [review]

**Pull request merged**